packages/server/migrations/20240321154546_create_team_members_table.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.hasTable('TeamMembers').then((exists) => {
```

Comment:
Do you guys really need this table? Maybe `employees` table could have another column.

Reply:
The purpose of this TeamMembers table is to keep records of the team members, as well as to make it easy to record if one person becomes a part of a different team in the future.

Reply:
Ok, I understand now.

Maybe we you guys need to add some foreign keys here as well 🤔

What do you think @nishadipri @SpartanDeBoss